specification/e-referrals-service-api.yaml

```diff
@@ -1292,6 +1292,39 @@ paths:
           $ref: 'components/schemas/responses/searchForService/200Response.yaml'
         '400':
           $ref: 'components/schemas/responses/searchForService/400Response.yaml'
+  /FHIR/STU3/ReferralRequest/{ubrn}/$ers.changeShortlist:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        To be added.....
+      
+      summary: Axxx - Change Shortlist
+      operationId: Axxx - Change Shortlist
+      parameters:
+        - $ref: 'components/schemas/headers/request/BearerAuthorization.yaml'
+        - $ref: 'components/schemas/headers/request/OdsCode.yaml'
+        - $ref: 'components/schemas/headers/request/BusinessFunction.yaml'
+        - $ref: 'components/schemas/headers/request/CorrelationID.yaml'
+        - $ref: 'components/schemas/headers/request/IfMatch.yaml'
+        - $ref: 'components/schemas/path/Ubrn.yaml'
+      requestBody:
+        required: true
+        content:
+          application/fhir+json:
+            schema:
+              $ref: 'components/schemas/eRS-ChangeShortlist-Parameters.yaml'
+            examples:
+              unbookedReferralRequest:
+                summary: Unbooked ReferralRequest
+                description: Changing the service list of an unbooked `ReferralRequest`
+                value:
+                  $ref: 'components/examples/changeShortlist/requests/UnbookedReferral.json'
+      responses:
```

Comment:
Shoudn't we document some error responses here? At least the once common to all endpoints ?

Reply:
We should yup, have added in the error codes now :)